Examples/ObjectiveC/ImageSourceExample.m

```diff
@@ -0,0 +1,46 @@
+
+#import "ImageSourceExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleImageSource = @"ImageSourceExample";
+
+@interface ImageSourceExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ImageSourceExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle darkStyleURL]];
+    mapView.delegate = self;
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(43.457, -75.789) zoomLevel:4 animated:NO];
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Set the coordinate bounds for the raster image.
+    MGLCoordinateQuad coordinates = MGLCoordinateQuadMake(
+                                                          CLLocationCoordinate2DMake(46.437, -80.425),
+                                                          CLLocationCoordinate2DMake(37.936, -80.425),
+                                                          CLLocationCoordinate2DMake(37.936, -71.516),
+                                                          CLLocationCoordinate2DMake(46.437, -71.516));
+    
+    // Create a MGLImageSource, which can be used to add georeferenced raster images the style of a map.
```

Comment:
`add georeferenced raster images the style of a map.`
👇 
`add georeferenced raster images to a map.`